.github/actions/setup-builder/action.yml

```diff
@@ -33,7 +33,6 @@ runs:
       run: |
         echo "Installing ${RUST_VERSION}"
         rustup toolchain install ${RUST_VERSION}
-        rustup override set ${RUST_VERSION}
```

Comment:
i **think** this is safe to remove, `./.github/actions/setup-builder` is only called once in github action workflows. So the `cargo` commands will default to the only toolchain thats installed. 

Except for in `ci.yml`, its called twice, to install nightly and MSRV. But in `make check-msrv`, we specify which toolchain to use.